bin/www

```diff
@@ -88,3 +88,11 @@ function onListening() {
     : 'port ' + addr.port;
   debug('Listening on ' + bind);
 }
+
+const mongoose = require('mongoose')
+mongoose.Promise = global.Promise
+const connection ='mongodb://localhost/lizardboard'
+
+mongoose.connect(connection)
+  .then(() => console.log('connection succesful'))
```

Comment:
Please do not add output here, but change this such that the server only starts in the event that we establish a connection to the database.  I think @harmanlearns had that in his team's PR last week (or at least got a comment with code samples)